tests/e2e-tests/specs/front-end/front-end-checkout.test.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import { activatePlugin } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { createSimpleProduct } from '../../utils/components';
+import { CustomerFlow, StoreOwnerFlow } from '../../utils/flows';
+import { setCheckbox, settingsPageSaveChanges, uiUnblocked, verifyCheckboxIsSet } from "../../utils";
+
+describe( 'Checkout page', () => {
+	beforeAll( async () => {
+		await activatePlugin( 'woocommerce' );
+		await createSimpleProduct();
+
+		// Go to general settings page
+		await StoreOwnerFlow.openSettings( 'general' );
```

Comment:
I think it is fine to merge this PR with this code to adjust the settings, but IMHO it would be better to have this configuration in another place. Especially considering that everything that is configured here is pretty standard. Maybe in a place that is executed only once as I mentioned in the comment above or in a base database file. I know we already talked a little bit about this and we disagree. I'm happy to proceed as is and then we can reevaluate in the future.

Reply:
We are now configuring initial settings in the `setup-wizard.test.js` test that runs before all other tests. However, I find it is still important to verify the settings in each test since they can be different for each test.